slack/slack.go

```diff
@@ -67,7 +69,14 @@ Loop:
 				readBotInfo(api)
 			case *slack.MessageEvent:
 				if !ev.Hidden && !ownMessage(ev.User) {
-					b.MessageReceived(ev.Channel, ev.Text, extractUser(ev.User))
+					ti, _ := api.GetTeamInfo()
```

Comment:
This team info could be obtained after connection to avoid making a request for each received msg couldn't it?